pkgs/development/python-modules/pytimeparse2/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, buildPythonPackage, dateutils }:
+
+buildPythonPackage rec {
+  pname = "pytimeparse2";
+  version = "1.7.1";
+
+  src = fetchFromGitHub {
+    owner = "onegreyonewhite";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-zWRbSohTvbVd3GcRRoxH/UReVGYHC0YmbNgbt8N0X48=";
+  };
+
+  propagatedBuildInputs = [ dateutils ];
+
+  # custom checks, see
+  # https://github.com/onegreyonewhite/pytimeparse2/blob/e00df7506b6925f2c6a5783e89e9f239d128271a/tox.ini#L36C20-L36C78
+  checkPhase = ''
+    runHook preCheck
+    python tests.py -vv --failfast
+    runHook postCheck
+  '';
+
+  pythonImportChecks = [ "pytimepase" ];
```

Comment:
Certainly a typo?

Reply:
oh boy, actually two typos. Its `pythonImportsCheck` and not `pythonImportChecks`. This is why it didn't catch the error in the first place.